### PR TITLE
Coin market cap

### PR DESCRIPTION
**Pull request by githubRaj**
Created the CoinMarketCap Repo as backup, as requested through issue #66 . Testing done on whether or not data is retrieved successfully as well, in accordance with the maintainability of the app (issue #39 ).

### CODE REVIEW

File: src/main/java/nitezh/ministock/dataaccess/CoinMarketCapRepo.java [hunk 1]
```diff
@@ -0,0 +1,37 @@
+package nitezh.ministock.dataaccess;
+
+/**
+ * Created by raj34 on 2018-04-04.
+ */
+
+
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
```
Comment (GurkomalSRao) [inline src/main/java/nitezh/ministock/dataaccess/CoinMarketCapRepo.java]: remove `import org.json.JSONObject;`
unnecessary import

File: src/test/java/nitezh/ministock/dataaccess/CoinMarketCapRepoTest.java [hunk 2]
```diff
@@ -19,11 +20,24 @@
     CoinMarketCapRepo coinMarketCapRepo;
 
     @Test
-    public void getAPIData() throws JSONException{
-        String id = "bitcoin";
+    public void getAllAPIData() throws JSONException{
+        String blankID = "";  //should get all the data if no id is provided
```
Comment (GurkomalSRao) [inline src/test/java/nitezh/ministock/dataaccess/CoinMarketCapRepoTest.java]: this only tests for top 100 cryptocurrencies. test whether you can retrieve all 1500 values

Add another test to validate each individual variable within the JSON Object/Array. You may retrieve bitcoin but the prices,marketcap or volume may be empty.

File: src/main/java/nitezh/ministock/dataaccess/CoinMarketCapRepo.java (hunk 1 above)
Comment (githubRaj) [reply src/main/java/nitezh/ministock/dataaccess/CoinMarketCapRepo.java]: Done

**Review by GurkomalSRao — CHANGES_REQUESTED** [review]
requested changes are in comments

**Review by GurkomalSRao — APPROVED** [review]

**Pull request merged**